azure_storage_client.go

```diff
@@ -204,7 +204,7 @@ func (util *snowflakeAzureClient) uploadFile(
 		})
 	} else {
 		var f *os.File
-		f, err = os.OpenFile(dataFile, os.O_RDONLY, os.ModePerm)
+		f, err = os.OpenFile(dataFile, os.O_RDONLY, 0666)
```

Comment:
What would you think about boy scout rule and replacing `os.OpenFile` with `os.Open`? We don't need permission here as we don't create a file.